draft-packages/collapsible/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "@kaizen/draft-collapsible",
+  "version": "1.0.0",
+  "description": "The draft Collapsible component",
+  "scripts": {
+    "prepublish": "tsc --project tsconfig.dist.json",
+    "build": "yarn clean && yarn prepublish",
+    "build:watch": "yarn clean && yarn prepublish --watch",
+    "clean": "rimraf '*.d.ts' '*.js' '*.map'"
```

Comment:
The only thing to be careful of is this won't clean those `KaizenDraft/thing.js` files... Did you manually remove them when you made your last commit?

Reply:
Changing all of these to 
```
"clean": "rimraf '*.d.ts' '*.js' '*.map' 'KaizenDraft/**/*.d.ts' 'KaizenDraft/**/*.js' 'KaizenDraft/**/*.map'"
```
should do the trick